zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/interceptors/getter/FsCacheGetterInterceptor.kt

```diff
@@ -0,0 +1,13 @@
+package app.cash.zipline.loader.interceptors.getter
+
+import app.cash.zipline.loader.ZiplineCache
+import app.cash.zipline.loader.ZiplineFile
+import app.cash.zipline.loader.ZiplineFile.Companion.toZiplineFile
+import okio.ByteString
+
+class FsCacheGetterInterceptor(
+  private val cache: ZiplineCache,
+  ): GetterInterceptor {
```

Comment:
I don’t think this is an interceptor because it doesn’t have a chaining feature . . . Maybe this is a reader?